composer.json

```diff
@@ -13,22 +13,17 @@
   ],
   "description": "Similar to js-husky client git hook implementation",
   "require": {
+    "php": ">=7.2",
```

Comment:
why ?

Reply:
Should we go for just `>=7` to support the PHP 7 major version?